framework/test/jest/integration/specs/modules/chain/components/storage/entities/round_delegates/summed_round.spec.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+const {
+	RoundDelegates,
+} = require('../../../../../../../../../../src/modules/chain/components/storage/entities');
+const { PgHelper } = require('../../../../../../../utils/pg-helper');
+
+describe('storage.entities.RoundDelegates.summedRound', () => {
+	let pgHelper;
+	let storage;
+	let db;
+
+	beforeAll(async () => {
+		// Arrange
+		pgHelper = new PgHelper({ dbName: 'RoundDelegatesSummedRound' });
+
+		// Create second postgres connection
+		db = await pgHelper.bootstrap();
+
+		// Setup storage for Accounts
+		storage = await pgHelper.createStorage();
+		storage.registerEntity('RoundDelegates', RoundDelegates);
+	});
+
+	afterAll(async () => {
+		await pgHelper.cleanup();
+	});
+
+	describe('Given arguments = (round, activeDelegates, tx)', () => {
+		it.skip('should return roundSummary', async () => {
```

Comment:
Still a skipped test here?

Reply:
as mentioned earlier I'll complete it with #4270. https://github.com/LiskHQ/lisk-sdk/pull/4269#discussion_r327492676